edb/schema/constraints.py

```diff
@@ -1068,8 +1068,14 @@ def _apply_field_ast(
                 )
             return
         elif op.property == 'args':
+            assert isinstance(node, qlast.ConstraintOp)
             assert isinstance(op.new_value, s_expr.ExpressionList)
-            node.args = [arg.qlast for arg in op.new_value]
+            args = []
```

Comment:
How does this help?

Reply:
`ConstraintOp.args` expects a `List[qlast.Expr]`, but `s_expr.Expression.qlast` is `qlast.Base`.  I decided to not ignore this.